crates/bevy_render/src/lib.rs

```diff
@@ -148,44 +162,43 @@ impl Plugin for RenderPlugin {
         .add_system_to_stage(CoreStage::PreUpdate, draw::clear_draw_system.system())
         .add_system_to_stage(
             CoreStage::PostUpdate,
-            camera::active_cameras_system.system(),
+            camera::active_cameras_system.system().label(RenderSystem::ClearDraw),
```

Comment:
Why does the system name and label differ so much here? Your name seems better, so perhaps `active_cameras_system` should be changed to match?

Reply:
It's because I was getting fancy with my editor and accidentally reverted a typo fix from a second window. The label should be on the system before the one it's on.